packages/rich-text-editor/src/styles/vaadin-rich-text-editor-popup-overlay-base-styles.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright (c) 2016 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { css } from 'lit';
+
+export const richTextEditorPopupOverlayStyles = css`
+  [part='overlay'] {
+    padding: var(--vaadin-rte-overlay-padding, var(--vaadin-padding-container));
+  }
+
+  [part='content'] {
+    display: grid;
+    gap: var(--vaadin-rte-overlay-gap, var(--vaadin-gap-container-inline));
+    grid-template-columns: repeat(7, minmax(0, 1fr));
+  }
+
+  [part='content'] ::slotted(button) {
+    border: var(
+      --vaadin-rte-overlay-button-border,
+      var(--vaadin-rte-button-border-width, 1px) solid var(--vaadin-rte-button-border-color, transparent)
+    );
+    border-radius: var(--vaadin-rte-overlay-button-border-radius, 9999px);
+    cursor: var(--vaadin-clickable-cursor);
+    font-size: var(--vaadin-rte-overlay-button-font-size, inherit);
+    height: var(--vaadin-rte-overlay-button-height, 1lh);
+    line-height: var(--vaadin-rte-overlay-button-line-height, inherit);
+    padding: var(--vaadin-rte-overlay-button-padding, 0);
+    width: var(--vaadin-rte-overlay-button-width, 1lh);
+  }
```

Comment:
These are for the color selection buttons, right? I wonder if another property name would be better then, to make it clear what they target, e.g. `--vaadin-rich-text-editor-color-option` or something like that.